nyc_data/ppe/data_mappings.py

```diff
@@ -11,7 +11,7 @@
 from xlsx_utils import SheetMapping, Mapping
 
 
-class DataType(str, Enum):
+class DataSource(str, Enum):
     EDC_PPE = "edc_ppe_data"
     EDC_MAKE = "edc_suppliers_partners"
     INVENTORY = "inventory"
```

Comment:
`edc_ppe_data`, `edc_suppliers_partners`, and `inventory` are just conventions we made up to account for different file names being uploaded, correct? In other words, we want to standardize the name of the `.xlsx` files that are getting imported, and made up these names to do that?

Reply:
Yeah totally made up. Need to be cleaned up and made sensible